docs/source/reference/api-only.md

```diff
@@ -1,37 +1,24 @@
-(api-only)=
```

Comment:
This `(api-only)=` line is a [MyST target](https://myst-parser.readthedocs.io/en/latest/syntax/syntax.html#targets-and-cross-referencing), allowing other documents to link to this document by referencing `api-only` independent of what the title is etc.

It won't be visible when this document is built from a Markdown (.md) file into a HTML file using the tool Sphinx and MyST-parser.

Reply:
Thank you! @consideRatio...I'll update it in my next PR

Reply:
Your'e welcome, thanks for your work!!

Note that you can update this PR by adding a commit. You could also replace the previous commit but that is good to avoid as then reviewers can loose track of what has changed since last inspection of a PR. When a commit is added as compared to replaced, its easy to see what has changed over time.